.github/workflows/test-coverage.yaml

```diff
@@ -0,0 +1,58 @@
+on:
```

Comment:
Is this based on https://github.com/r-lib/actions/blob/master/examples/test-coverage.yaml? If so, please add a comment, like in the other `.yaml` file.